frontend/src/views/create/index.tsx

```diff
@@ -3,26 +3,46 @@ import { Link, useNavigate } from 'react-router-dom';
 import Arrow from '../../assets/icon/arrow-left-solid.svg';
 import { stockData } from '../../data';
 import { Book } from '../../types';
-import { ENDPOINTS } from '../../utils/endpoint';
 import { useCreateBook } from './hook/useCreateBook';
 
 /**
  * Create component that renders a form for creating a new book.
  *
- * @component
+ * @returns {React.ReactElement} A react component that renders a form for creating a new book,
+ * it allows to submit a form and handle errors if they occurs.
+ *
  * @example
- * return (
  *  <Create />
- * )
  */
 const Create: React.FC<{}> = (): React.ReactElement => {
   const navigate = useNavigate();
-  const { isError, isLoading, sendData } = useCreateBook(ENDPOINTS.BOOKS);
+  const { error, isLoading, sendData } = useCreateBook();
+
+  /**
+   * Handle the submit of the form.
+   *
+   * @param {Object} values - The values of the form, it should contain the properties of a book object, except the id.
+
+   * @returns {void}
+   */
+  const handleSubmit = (values: Omit<Book, 'id'>) => {
+    createBook(values);
+  };
 
-  const createBook = (values: Omit<Book, 'id'>) => {
-    sendData(values).then(() => navigate('/', { replace: true }));
+  /**
+   * Create a new book by sending the form data to the server and navigate to homepage.
+   * @async
+   *
+   * @param {Object} values - The values of the form, it should contain the properties of a book object, except the id.
+   *
+   */
+  const createBook = async (values: Omit<Book, 'id'>) => {
+    await sendData(values);
+    navigate('/', { replace: true });
   };
 
+  if (error.isError) return <div>{error.message}</div>;
```

Comment:
What happens if there is an error? How the user can recover the app or if we want the error inside a popup? Think about it ;)